src/App.js

```diff
@@ -0,0 +1,62 @@
+import './style/Global.css';
+import Nav from './components/Nav';
+import Profile from './pages/Profile';
+import { Home } from './pages/Home';
+import { NotFound } from './pages/NotFound';
+import { Resources } from './pages/Resources';
+import Footer from './components/Footer';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import { AuthenticatedComponent, useAuthContext } from '@asgardeo/auth-react';
+import { Settings } from './pages/Settings';
+
+function App() {
+
+    const { state } = useAuthContext();
```

Comment:
Check the indentation.

Reply:
Fixed by a66cbb117ca5bd1bc65d7fff6d176b7a71b932b9.